lib/std/ascii.zig

```diff
@@ -9,6 +9,48 @@
 
 const std = @import("std");
 
+/// Contains constants for the C0 control codes of the ASCII encoding.
+/// https://en.wikipedia.org/wiki/C0_and_C1_control_codes
+pub const control_code = struct {
```

Comment:
This might make sense as an enum?

Reply:
I thought so at first, but having constants for them makes more sense both from a semantic and a usability perspective IMO — there's nothing like "several variants of one overarching C0 control code concept" that you *could* enumerate, just different codes under a common heading; also having it as an enum would mandate `@enumToInt` on every use case for these that I can imagine.

Reply:
@daurnimator i thought so first, too. but when doing an enum, it should contain all valid ASCII codes to be really usable, exhaustive enum wouldn't just cut it (because then i would have to use a lot of `@enumToInt` and `@intToEnum`)

as @deingithub says: it makes more sense as a set of named constants, way easier to handle